packages/dashboard/src/vaadin-dashboard.js

```diff
@@ -195,6 +195,12 @@ class Dashboard extends DashboardLayoutMixin(ElementMixin(ThemableMixin(PolylitM
           };
         },
       },
+
+      /* @private */
```

Comment:
nit: comment syntax.

```suggestion
      /** @private */
```

Reply:
Done